client/my-sites/email/email-management/email-home.jsx

```diff
@@ -1,203 +0,0 @@
-import { Card } from '@automattic/components';
-import { localize } from 'i18n-calypso';
-import page from 'page';
-import PropTypes from 'prop-types';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import titleCase from 'to-title-case';
-import DocumentHead from 'calypso/components/data/document-head';
-import QuerySiteDomains from 'calypso/components/data/query-site-domains';
-import EmptyContent from 'calypso/components/empty-content';
-import Main from 'calypso/components/main';
-import SectionHeader from 'calypso/components/section-header';
-import { hasEmailForwards } from 'calypso/lib/domains/email-forwarding';
-import { hasGSuiteWithUs } from 'calypso/lib/gsuite';
-import { hasTitanMailWithUs } from 'calypso/lib/titan';
-import EmailHeader from 'calypso/my-sites/email/email-header';
-import EmailListActive from 'calypso/my-sites/email/email-management/home/email-list-active';
-import EmailListInactive from 'calypso/my-sites/email/email-management/home/email-list-inactive';
-import EmailNoDomain from 'calypso/my-sites/email/email-management/home/email-no-domain';
```

Comment:
Did you check all those imports? As far I can see this component is no longer used, and should be removed.

Reply:
Good catch! I didn't check until now, but those components are still in use! :)

Reply:
EmailHeader is quite common and frequently used. The other 3 are being used in the Typed version of this component.

Reply:
Sorry, I'm not sure how I've messed this up. I've looked up `EmailNoDomain` and couldn't find anything, and even double checked since I found that surprising.